lib/ast.js

```diff
@@ -1114,6 +1113,14 @@ var AST_Number = DEFNODE("Number", "value literal", {
     }
 }, AST_Constant);
 
+var AST_BigInt = DEFNODE("BigInt", "value literal", {
+    $documentation: "A big int literal",
+    $propdoc: {
+        value: "[number] the numeric value",
```

Comment:
A `number` cannot accurately represent the value of a big integer. The node's value should be a `BigInt`.

Reply:
May be `string` ?